src/main/java/org/junitpioneer/jupiter/issue/IssueExtensionExecutionListener.java

```diff
@@ -37,7 +36,7 @@
  */
 public class IssueExtensionExecutionListener implements TestExecutionListener {
 
-	public static final String REPORT_ENTRY_KEY = "Issue";
+	public static final String REPORT_ENTRY_KEY = "IssueExtensionExecutionListener";
```

Comment:
This key is used by `IssueExtension` to publish report entries. Why would it use the key of the "target" of the message. If "Issue" is too general, let's stick to "Issue extension" (or "IssueExtension"`).